wire-ios-sync-engine/Source/SessionManager/SessionManager.swift

```diff
@@ -452,7 +455,8 @@ public final class SessionManager: NSObject, SessionManagerType {
          isUnauthenticatedTransportSessionReady: Bool = false,
          sharedUserDefaults: UserDefaults,
          minTLSVersion: String? = nil,
-         deleteUserLogs: (() -> Void)? = nil
+         deleteUserLogs: (() -> Void)? = nil,
+         analyticsSessionConfiguration: AnalyticsSessionConfiguration? = nil
```

Comment:
I suggest to not provide a default value.
```suggestion
         analyticsSessionConfiguration: AnalyticsSessionConfiguration?
```